scripts/f5-config.py

```diff
@@ -14,31 +14,57 @@
 # limitations under the License.
 #
 # (c) 2014, Kevin Carter <kevin.carter@rackspace.com>
+from __future__ import print_function
 import argparse
 import json
 import os
+import sys
 
 import netaddr
 
+IS_LEM = True
 
-PART = 'RPC'
-PREFIX_NAME = 'RPC'
+if 'LAB_DATA_DIR' in os.environ:
+    # If we have LAD_DATA_DIR set, we are operating inside a lab environment
+    LAB_DATA_DIR = os.environ['LAB_DATA_DIR']
```

Comment:
This suggested edit looks logical.  I also think you should print the 'Not a LEM' thing to stderr, because the script takes a --print arg so you can print the F5 to stdout, and starting to throw in non-config stuff into that output could break stuff.